workshops/JamBlog/README.md

```diff
@@ -0,0 +1,453 @@
+
+## The Stack
+
+### [Jamstack](https://jamstack.org/what-is-jamstack/)
+
+You've probably heard of the Jamstack or come across it a few times,
+Jamstack is the new standard architecture for the web designed to make websites faster. Using Git workflows and modern build tools, pre-rendered content is served to a **CDN** and made dynamic through **APIs** and **serverless functions**. Technologies in the stack include JavaScript frameworks, Static Site Generators, Headless CMS, and CDNs.
+
+Technologies in this workshop:
+
+- React.js(Javascript framework)
+- Next.js(Static site Generator)
+- Ghost.js(Headless CMS)
+
+Hosting on a CDN is beyond the scope of this workshop.
+
+### [Next.JS](https://nextjs.org/)
+
+Next.js is an open-source React front-end development web framework that enables functionality such as server-side rendering and generating static websites for React based web applications. It is a production-ready framework that allows developers to quickly create static and dynamic Jamstack websites.
+
+### Headless CMS
+
+An headless CMS(content management system), is a back-end only content management system (CMS) built from the ground up as a content repository that makes content accessible in an API driven manner via a RESTful API for display on any client, regardless of the device.
+
+So it’s basically a full-backend service with the database and also the editor and everything set up for you without any view representation, you interact with the API and JSON response. There are several headless cms examples out there, Including Ghost.js I’ll show an example of how to build a blog feature in a NextJS website with Ghost.js headless cms. This is what we’re going to develop in this workshop.
+
+### why [Ghost.js](https://ghost.org/)
+
+Ghost is the number one open-source CMS on Github. Ghost is an open-source, professional publishing platform built on a modern Node.js technology stack, and is completely framework-agnostic — designed with flexibility and performance built-in.
+
+Everyday Ghost powers some of the most-read stories on the internet, serving hundreds of millions of requests across tens of thousands of sites.
+
+![we move](./img/we_move.gif)
+
+###### we move, one step at a time...
+
+## Prerequisites
+
+This configuration of a Ghost publication requires an existing moderate knowledge of JavaScript and React. but everybody is welcome to follow along, I would try to break stuff down as much as possible.
+
+![giphy](./img/dance.gif)
+
+###### _Let's get this party started_
+
+## Setup Ghost.js
+
+Ghost is completely free and open-source. It can be downloaded and self-hosted just about anywhere or spun up in a few moments using the Ghost(Pro) managed PaaS, but this includes a subscription which comes with a 14-day free trial.
```

Comment:
I think you should remove this paragraph. This information isn't directly relevant to the workshop